index.html

```diff
@@ -46,778 +46,778 @@ <h3>Spring 2023, Class: Mon, Wed 4:30pm-5:50pm, Gates B1</h3>
-					qualified educational expenses for tax purposes. If you are an undergraduate receiving financial
-					aid, you may be eligible for additional financial aid for required books and course materials if
-					these expenses exceed the aid amount in your award letter. For more information, review your award
-					letter or visit the <a href="https://financialaid.stanford.edu/undergrad/budget/index.html">Student
-						Budget website</a>. </p>
-			</div>
-
-
-			<br>
-
-
-			<!-- The footer -->
-			<div class="hrline">
-				<hr>
-			</div><br>
-			<div class="row clearfix">
-				<div class="col-md-3 column">
-				</div>
-				<div class="col-md-6 column">
-					<div class="text-center">
-						<p>
-							&nbsp;&nbsp;&nbsp; &#169; Chelsea Finn 2023
-						</p>
-					</div>
-				</div>
-				<div class="col-md-3 column">
-				</div>
-			</div>
-			<br>
-			<br>
-			<br>
-			<!-- End of document -->
-		</div>
+                        </td>
+                        <td>
+                            <ul>
+                                <li> <a href="https://link.springer.com/article/10.1007/BF00992696"> Simple statistical
+                                        gradient-following algorithms for connectionist reinforcement learning.</a>
+                                    Williams (1992)</li>
+                            </ul>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td><b>Week 2</b><br />
+                            Fri, April 11
+                        </td>
+                        <td><i><span class="label label-lecture text-base">Lecture</span></i>
+                            <a">Actor-Critic Methods</a>
+                        </td>
+                        <td>
+                        </td>
+                        <td>
+                            <ul>
+                                <li> <a
+                                        href="https://proceedings.neurips.cc/paper_files/paper/1999/file/464d828b85b0bed98e80ade0a5c43b0f-Paper.pdf">
+                                        Policy Gradient Methods for Reinforcement Learning with Function Approximation.
+                                    </a> Sutton et al. (1999)</li>
+                                <li> <a href="https://arxiv.org/pdf/1707.06347"> Proximal
+                                        Policy Optimization.</a> Schulman et al. (2017)</li>
+                            </ul>
+                        </td>
+                    </tr>
+                    <tr class="active">
+                        <td><b>Week 3</b><br />
+                            Wed, April 16
+                        </td>
+                        <td><i><span class="label label-lecture text-base">Lecture</span></i>
+                            <a">Q-Learning</a>
+                        </td>
+                        <td>
+                            <span class="label label-due text-base">Due</span></i> <b>Survey Due</b><br>
```

Comment:
could we edit this to "Project Survey"? (note that you also don't need the "due" text because of the red label.